Docs/user_guide/get_help.md

```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
+  - Type your queston into Google. There is a good chance that your question has been asked and anwered before and all questions ever asked about Slicer are publicly available and indexed by Google.
```

Comment:
Would it be better to search the question into the Slicer Discourse forum directly instead of into Google?  Google might provide unrelated results though I guess the question could be not specific to Slicer either.